operator/src/main/resources/scripts/modelInImage.sh

```diff
@@ -53,6 +53,11 @@ RCU_PASSWORD_CHANGED=5
 SCRIPT_ERROR=255
 
 export WDT_MODEL_SECRETS_DIRS="/weblogic-operator/config-overrides-secrets"
+if [ ! -d ${WDT_MODEL_SECRETS_DIRS} ] ; then
+  # Temporary work around for WDT to avoid non zero return code if the directory doesn't exist
+  mkdir -p /tmp/nosuchdir
+  export WDT_MODEL_SECRETS_DIRS="/tmp/nosuchdir"
+fi
```

Comment:
Does it suppress the WARNING if you skip the export all-together?

Does this need to be a temporary work-around?  The  WARNING is distracting in the first place, even if WDT is fixed to return non-zero.   Maybe just remove the comment. 

If answer to both is yes, the change is as sample as:
[ ! -d ${WDT_MODEL_SECRETS_DIRS} ]  && unset WDT_MODEL_SECRETS_DIRS

Reply:
I requested a fix on WDT to make it INFO instead of WARNING and remove the default `weblogic-operator/config-overrides-secrets`  Once WDT fixes and release, we shouldn't need this all together.

Reply:
OK. That's fine.